manager/src/main/java/com/redhat/service/bridge/manager/ProcessorServiceImpl.java

```diff
@@ -121,23 +121,22 @@ public Processor createProcessor(String bridgeId, String customerId, ProcessorRe
         ProcessorDefinition definition = new ProcessorDefinition(requestedFilters, requestedTransformationTemplate, requestedAction, resolvedAction);
         newProcessor.setDefinition(definitionToJsonNode(definition));
 
-        LOGGER.info("Processor with id '{}' for customer '{}' on bridge '{}' has been marked for creation",
-                newProcessor.getId(),
-                newProcessor.getBridge().getCustomerId(),
-                newProcessor.getBridge().getId());
-
         createProcessorConnectorEntity(newProcessor, actionProvider, resolvedAction);
 
-        workManager.schedule(newProcessor);
-
         return newProcessor;
     }
 
     @Transactional
-    // Connector should always be marked for creation in the same transaction as a Processor
+    // Processor, Connector and Work should always be created in the same transaction
```

Comment:
Same general concepts here about method consolidation as in `BridgeServiceImpl` 👍

Reply:
@robpblake Updates completed.